addons/l10n_in/views/report_invoice.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="l10n_in_report_invoice_document_inherit" inherit_id="account.report_invoice_document">
+        <xpath expr="//div[@id='total']" position="after">
+            <div id="total_in_words" class="row mb16">
+                <div class="col-xs-2">
+                    <strong>Total (In Words): </strong>
+                </div>
+                <div class="col-xs-8">
+                    <span t-field="o.amount_total_words"/>
+                </div>
+            </div>
+        </xpath>
+        <xpath expr="//tbody[hasclass('invoice_tbody')]/tr/td[1]" position="replace">
```

Comment:
I might not have been clear: I'm fine with replacing the whole table for the invoice lines, I just don't want a brand new report. Can you do so?